spec/support/container_helpers.rb

```diff
@@ -232,4 +232,50 @@ def uuid_regex
       cgroup_file.rewind
     end
   end
+
+  shared_context 'Fargate 1.4+ (2-part short random) environment' do
+    include_context 'cgroup file'
+
+    let(:platform) { 'ecs' }
+    let(:container_id_with_random) { "#{container_id_without_random}-609015642" }
+    let(:container_id_without_random) { 'cef584f232933b25e0c6933d7e86cb34' }
```

Comment:
Sorry -- just checking, the only difference between this shared context and the `Fargate 1.4+ (2-part) environment` that precedes it, is that the `container_id_with_random` after the `-` has 9, instead of 10 characters, is that it?

If so, I would suggest either leaving a comment, or perhaps adding that part as a let to call it out in a more obvious way, e.g. `let(:id_suffix_smaller_than_10_chars) { '...' }` / `let(:container_id_with_random) { "#{container_id_without_random}-#{id_suffix_smaller_than_10_chars}"`.

Reply:
I described it as `short random` :)
But I can add more context.